queries/lua/textobjects.scm

```diff
@@ -76,8 +76,8 @@
  (#make-range! "parameter.outer" @parameter.inner @_end))
 
 (table_constructor
-  . (_) @parameter.inner
-  . ","? @_end
+  (field) @parameter.inner
+  ","? @_end
```

Comment:
Any pairs of `field` and `,` will create a mapping. I don't think that's intended?

Reply:
If anything, the original version should be updated to 
```
(table_constructor
  . (comment)*
  . (field)
  . ",")
```

Reply:
I see what you are saying. I was wondering about this, but it seems that even the general capture works properly (searching the closest comma-field pairing). I went with this query group because it simplified things, I think for complete correctness the other query would have to be:
```query
(table_constructor
  . (comment)*
  . (field)
  . (comment)* ; potential inline comments
  . ",")
```
This would rarely (if ever) happen, but it still could, and this query looks a lot weirder than the other simpler one that still works despite capturing maybe many more pairings. Unless you are saying that the captures themselves are an issue (maybe speed-wise?)

Reply:
Unfortunately that query starts breaking once you try to put captures in, and you end up having to do something a little less elegant to cover these cases, like I did in the aforementioned draft PR